### ISSUE TITLE
Cant get location problem

### ISSUE THREAD

**Issue by tahakuzgun** [kernel65/Auto-Run-1.1#2]
Hello, thank you for the video you added.
I ran it with visual code. I added it to Overwolf as you wrote.
But it cannot fetch the location as in the video. It says 0 0 0. That's why I can't start it successfully.
Ex: Current farm point is 0 0 0
In the video, I saw a log that records the location in a few seconds. Unfortunately it doesn't happen to me.

**Comment by tahakuzgun** [comment]
I made a 1 minute video. I hope it helps.
https://streamable.com/ymnkt0

**Comment by Amir-Ebrahimzadeh** [comment]
Hello, I get this error. Do you know what should i do?
System.UnauthorizedAccessException: 'Access to the path 'C:\Users\Asus\Documents\OverwolfAutopath\stream.pos' is denied.'

**Comment by kernel65** [comment]
Hello Amir, you need to create stream.pos file in this folder. If you haven't OverwolfAutopath folder, create it too.

**Comment by kernel65** [comment]
> tahakuzgun

Hello, you need to be whitelisted in Overwolf like a developer.

**Comment by Amir-Ebrahimzadeh** [comment]
I have created that file

> Hello Amir, you need to create stream.pos file in this folder. If you haven't OverwolfAutopath folder, create it too.

**Comment by Amir-Ebrahimzadeh** [comment]
> Hello Amir, you need to create stream.pos file in this folder. If you haven't OverwolfAutopath folder, create it too.

Thanks for help.

**Comment by Amir-Ebrahimzadeh** [comment]
Can i add wirefiber or fiber to bot?

**Comment by kernel65** [comment]
> Can i add wirefiber or fiber to bot?

Yes. Firstly you need to modify TEMPLATES massive in FileManager.cs file, after that compile project in Visual Studio. Secondly make a template picture, check video in Readme please.

**Comment by kernel65** [comment]
How to modify TEMPLATES massive example - just add created picture name file ( template_fiber.png )    like @"Images\Templates\template_fiber.png"

Final version:
public static String[] TEMPLATES =
        {
            @"Images\Templates\template_blightcrag.png",
            @"Images\Templates\template_blightroot.png",
            @"Images\Templates\template_boulder.png",
            @"Images\Templates\template_bulrush.png",
            @"Images\Templates\template_bumbleblossom.png",
            @"Images\Templates\template_bush.png",
            @"Images\Templates\template_carrots.png",
            @"Images\Templates\template_dragonglory.png",
            @"Images\Templates\template_earth_mote.png",
            @"Images\Templates\template_earthcribe.png",
            @"Images\Templates\template_earthspine.png",
            @"Images\Templates\template_flint.png",
            @"Images\Templates\template_fronded_petalcap.png",
            @"Images\Templates\template_gold.png",
            @"Images\Templates\template_hemp.png",
            @"Images\Templates\template_herbs.png",
            @"Images\Templates\template_iron_vein.png",
            @"Images\Templates\template_lifebloom.png",
            @"Images\Templates\template_lighteeng_bee.png",
            @"Images\Templates\template_lodestone.png",
            @"Images\Templates\template_mature_tree.png",
            @"Images\Templates\template_orichalcum_vein.png",
            @"Images\Templates\template_platinum.png",
            @"Images\Templates\template_provisions_crate.png",
            @"Images\Templates\template_rivercress.png",
            @"Images\Templates\template_seeping_stone.png",
            @"Images\Templates\template_shockbulb.png",
            @"Images\Templates\template_silkweed.png",
            @"Images\Templates\template_silver.png",
            @"Images\Templates\template_soul_mote.png",
            @"Images\Templates\template_soulspire.png",
            @"Images\Templates\template_soulsprout.png",
            @"Images\Templates\template_soulwyrm.png",
            @"Images\Templates\template_suncreeper.png",
            @"Images\Templates\template_supply_crate.png",
            @"Images\Templates\template_young_tree.png",
            @"Images\Templates\template_fiber.png",
        };

**Comment by kernel65** [comment]
Later I will add the option to add templates through the program interface.

**Comment by Amir-Ebrahimzadeh** [comment]
Thank you for your response.

**Comment by Amir-Ebrahimzadeh** [comment]
I took a picture of wirefiber and crop it and put it on images folder and named it template_wirefiber.png , then added it in FileManager.cs as @"Images\Templates\template_wirefiber.png" and in Form1.Designer.cs as "Wirefiber" but when i want to farm it, it says farm point is emty.

**Comment by Amir-Ebrahimzadeh** [comment]
I did everything you said, I also added Wirefiber into Form1.Designer.cs but when i want to farm wirefiber it says Farm point is empty, go to next point

**Comment by kernel65** [comment]
You need to add items to the end of list in Form and Array TEMPLATES. The elements have a strict sequence, if you add a new element to the beginning of the array and in the Form interface to the end, they will not match each other.
